Update the README to point to InApp Messaging. (#115)
forcedotcom/ServiceSDK-iOS: README.md

# Second Generation Salesforce Chat Product

> Messaging for InApp, the modern replacement for ServiceChat

Reviewer: ```suggestion
Check out Messaging for In-App, a newer chat solution for mobile apps.
```

> [To Learn More](https://developer.salesforce.com/docs/service/messaging-in-app/overview)

Reviewer: ```suggestion
To learn more, see the [Messaging for In-App Developer Guide](https://developer.salesforce.com/docs/service/messaging-in-app/overview)
```

> [Test App Repo](https://github.com/Salesforce-Async-Messaging/messaging-in-app-ios/tree/master/examples)

Reviewer: ```suggestion
Try out an example from our [Sample App Repo](https://github.com/Salesforce-Async-Messaging/messaging-in-app-ios/tree/master/examples)
```
